misc/install-deps.sh

```diff
@@ -2,13 +2,17 @@
 
 if [ "x$(id -u)" != x0 ]; then
     echo "You might have to run it as root user."
-    echo "Please run it again with 'sudo'."
-    echo
+    echo "Please run it again with 'sudo'.\n"
```

Comment:
This requires `-e` option.

```
$ echo "abc\n"
abc\n

$ echo -e "abc\n"
abc

```

Reply:
I think in order to use `echo -e` in shell script, 
shebang has to be `/bin/bash`, not (`/bin/sh`)

Otherwise, `-e` is also printed as follows:
(echo consider -e as a string)

```bash
#!/bin/sh

if [ "x$(id -u)" != x0 ]; then
    echo "You might have to run it as root user."
    echo -e "Please run it again with 'sudo'.\n"
    exit
fi
```

```
~/uftrace/misc$ ./install-deps.sh
You might have to run it as root user.
-e Please run it again with 'sudo'.
```

So I think these are the solution candidates for this problem:
1. keep shebang as /bin/sh && using empty echo instead of \n
2. chang shebang as /bin/bash && use `echo -e "...\n"`

reference:
[1] https://stackoverflow.com/questions/36531795/echo-e-is-not-working-in-my-script
[2] https://stackoverflow.com/questions/10267852/echo-e-working-in-terminal-but-not-in-bash-script

Reply:
Oh, I didn't know that it's a bash thing.  Then I think we should keep the empty echo line (if you really want a blank line).

Reply:
OK. Let's keep using empty echo for newline again.

Reply:
I just changed last newline as `echo`. Thank you for your feedbacks :)